src/Forms/ListboxField.php

```diff
@@ -165,6 +166,17 @@ public function getDisabledItems()
         return $this->disabledItems;
     }
 
+    public function getValueForValidation(): mixed
+    {
+        // Unlike other MultiSelectField's, ListboxField allows setting a single value
+        // Wrap single values in an array so that the validation logic can treat it as a multi-select
```

Comment:
All `MultiSelectField` allows a single value, don't they? There's nothing that requires 2+ values to my knowledge.

Reply:
Correct, though it will be a single value wrapped in an array.  Easier than having to account for a value that may or may not be in an array